deployment/cfn/worker.py

```diff
@@ -417,7 +417,10 @@ def get_cloud_config(self):
                 '  - path: /etc/mmw.d/env/ROLLBAR_SERVER_SIDE_ACCESS_TOKEN\n',
                 '    permissions: 0750\n',
                 '    owner: root:mmw\n',
-                '    content: ', self.get_input('RollbarServerSideAccessToken')]  # NOQA
+                '    content: ', self.get_input('RollbarServerSideAccessToken'),  # NOQA
+                '\n',
+                'runcmd:\n',
+                '  - /opt/model-my-watershed/scripts/ebs-warmer.sh /dev/xvdf']
```

Comment:
We should monitor the output of this command during testing to get a benchmark for how long this process takes.